BHoM_UI/Caller/Run.cs

```diff
@@ -125,6 +125,13 @@ protected virtual List<object> CollectInputs()
                         try
                         {
                             input = m_CompiledGetters[i](m_DataAccessor, index);
+
+                            if (input == null || input.ToString() == "")
```

Comment:
wonder if this instead of checking for null should be checking some equality of default(T) or similar, as this would always get hit for numeric values etc.

Reply:
Replying myself here after another thought. My suggestion of the above would mean raising notes if a 0 value has been provided, which for many cases is very valid, so unless we move where the check is made to check if any value has been provided, would rather keep this as is.

Reply:
@IsakNaslundBh . Yes the reasoning was exactly around the `0` value. We always add a extra `OR` check later if we missed something